asf_core_data/pipeline/preprocessing/feature_engineering.py

```diff
@@ -338,14 +342,32 @@ def get_heating_features(df, fine_grained_HP_types=False):
         (df["HP_INSTALLED"])
         | (df["SECONDHEAT_DESCRIPTION"].str.lower().str.contains("heat pump"))
         | (df["MAINHEAT_DESCRIPTION"].str.lower().str.contains("pumpa teas"))
+        | (df["SECONDHEAT_DESCRIPTION"].str.lower().str.contains("pumpa teas"))
         | (df["MAINHEAT_DESCRIPTION"].str.lower().str.contains("pwmp gwres"))
+        | (df["SECONDHEAT_DESCRIPTION"].str.lower().str.contains("pwmp gwres"))
         | (df["MAINHEAT_DESCRIPTION"].str.lower().str.contains("bwmp gwres"))
+        | (df["SECONDHEAT_DESCRIPTION"].str.lower().str.contains("bwmp gwres"))
         | (df["MAINHEAT_DESCRIPTION"].str.lower().str.contains("pympiau gwres"))
-        | (df["MAINHEAT_DESCRIPTION"].str.lower().str.contains("bympiau gwres")),
+        | (df["SECONDHEAT_DESCRIPTION"].str.lower().str.contains("pympiau gwres"))
+        | (df["MAINHEAT_DESCRIPTION"].str.lower().str.contains("bympiau gwres"))
+        | (df["SECONDHEAT_DESCRIPTION"].str.lower().str.contains("bympiau gwres")),
```

Comment:
The search for HP in other languages should be both in `MAINHEAT_DESCRIPTION` AND `SECONDHEAT_DESCRIPTION`